### PR TITLE
Fix node tty.d.ts WriteStream.cursorTo function signature

### PR DESCRIPTION
**Pull request by pbelbin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [https://nodejs.org/api/tty.html#tty_writestream_cursorto_x_y_callback](https://nodejs.org/api/tty.html#tty_writestream_cursorto_x_y_callback)
- [] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Review by trivikr — APPROVED** [review]
Confirmed https://nodejs.org/api/tty.html#tty_writestream_cursorto_x_y_callback

**Review by SimonSchick — APPROVED** [review]

File: types/node/tty.d.ts
Comment (Flarna) [inline types/node/tty.d.ts]: please add also `cursorTo(x: number, callback?: () => void): boolean`.

Comment (pbelbin) [reply types/node/tty.d.ts]: Done, and added a test too.

**Review by Flarna — APPROVED** [review]

**Comment by Flarna** [comment]
@pbelbin CI fail is unrelated but maintainers won't merge a PR with red CI.
I recommend to retrigger by closing/reopening the PR or push a new commit (e.g. rebase to master,..).

**Pull request merged**
